packages/hub/routes/scheduled-payments.ts

```diff
@@ -127,8 +127,8 @@ export default class ScheduledPaymentsRoute {
       privateMemo: attrs['private-memo'],
     } as unknown as ScheduledPayment;
 
-    if (params.recurringDayOfMonth != null) {
-      params.payAt = calculateNextPayAt(new Date(), params.recurringDayOfMonth);
+    if (params.recurringDayOfMonth != null && params.recurringUntil != null) {
```

Comment:
Maybe we can add a default value to `recurringUntil` in case it is not defined? I.e one year from now like in the interface

Reply:
I think `recurringUntil` is mandatory for recurring payments.

Reply:
Added the validation https://github.com/cardstack/cardstack/pull/3718/commits/77c3846608277f7400875dc1f944e47a86de5991